cornedbeef/src/main/java/com/swiftkey/cornedbeef/PunchHoleCoachMark.java

```diff
@@ -88,13 +117,70 @@ protected void updateView(CoachMarkDimens<Integer> popupDimens,  CoachMarkDimens
                 horizontalPadding, verticalPadding + lowerGap);
     }
 
+    /**
+     * Move the punch hole from left to right of the target view, unless the
```

Comment:
The comment needs to be updated as now it's start to end and back from end to start. It'd be better to avoid using left and right (and I know I wrote that myself hehe).